.gitignore

```diff
@@ -1,3 +1,6 @@
 *.pyc
 */__pycache__
-psqldump
\ No newline at end of file
+psqldump
+*/.DS_Store
+musicbrainz/*
```

Comment:
I'm still confused why you are ignoring everything in the `musicbrainz` folder!

Reply:
I changed it before creating a new branch and I did not want the changes to reflect on this branch. I fixed it now.